### DISCUSSION TITLE
请问Table的分页要如何将 "条/页" 移除?

### DISCUSSION THREAD

**Discussion by kevintsai1202** [ant-design/ant-design#43898] [Q&A]
请问表格中选择每页几笔的下拉菜单要如何将 "条/页" 拿掉?我只想显示数字
![圖片](https://github.com/ant-design/ant-design/assets/12706683/ff8b708b-94fb-441d-bd07-d05ceb61636c)

**Comment by li-jia-nan** [comment]
`Pagination` 组件应该有 `itemRender` 属性吧，可以自定义渲染内容

**Comment by kevintsai1202** [comment]
找到了,在pagination下加上locale, 裡面有個items_per_page設成空字串就只會顯示數字了

 pagination={{
          showSizeChanger: true,
          locale: {
            items_per_page: ""
          }
        }}

**Reply by li-jia-nan** [reply]
啊……居然还可以这样……
